Resources/Prototypes/EstacaoPirata/Entities/Mobs/Customization/cyberlimbs/hesphiastos.yml

```diff
@@ -0,0 +1,115 @@
+- type: marking
```

Comment:
We need to move these down into the base folders, since these are going to be maintained here. Essentially instead of having Resources/Prototypes/EstacaoPirata/Entities/...
It just needs to be Resources/Prototypes/Entities/...